constraint/pkg/client/rego_helpers.go

```diff
@@ -0,0 +1,95 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/open-policy-agent/opa/ast"
+)
+
+// ensureRegoConformance rewrites the package path and ensures there is no access of `data`
+// beyond the whitelisted bits. Note that this rewriting will currently modify the Rego to look
+// potentially very different from the input, but it will still be functionally equivalent.
+func ensureRegoConformance(kind, path, rego string) (string, error) {
+	if rego == "" {
+		return "", errors.New("Rego source code is empty")
+	}
+	module, err := ast.ParseModule(kind, rego)
+	if err != nil {
+		return "", err
+	}
+	if err := rewritePackage(path, module); err != nil {
+		return "", err
+	}
+	if len(module.Imports) != 0 {
+		return "", errors.New("Use of the `import` keyword is not allowed")
+	}
+	if err := checkDataAccess(module); err != nil {
+		return "", err
+	}
+	return module.String(), nil
+}
+
+// rewritePackage rewrites the `package` statement in Rego with the provided package path
+func rewritePackage(path string, module *ast.Module) error {
+	if module.Package == nil {
```

Comment:
Nit: we could construct a new ast.Package here instead of requiring the user provide one only to overwrite it.

Reply:
done